### ISSUE TITLE
Error connect: connection refused when running make docker

### ISSUE THREAD

**Issue by Willem456** [istio/istio#37404]
### Bug Description

When I run make docker it shows an error like the following
![image](https://user-images.githubusercontent.com/74516404/154427846-c6c8bbb5-665e-48b4-be59-a9570cd3d2d3.png)
When I pull the image with docker it is successful
![image](https://user-images.githubusercontent.com/74516404/154428631-c5bc76e2-f752-47d5-9c6d-3ec8c36f3592.png)
Can someone help with this, thanks a lot

### Version

```prose
# kubectl version --short
Client Version: v1.20.6
Server Version: v1.20.6
```

### Affected product area

- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Upgrade
- [ ] Multi Cluster
- [ ] Virtual Machine
- [ ] Control Plane Revisions

### Is this the right place to submit this?

- [X] This is not a security vulnerability
- [X] This is not a question about how to use Istio

**Comment by ericvn** [comment]
What platform are you running on? What version of Docker/Docker Desktop are you using?

**Comment by Willem456** [comment]
@ericvn I running on CentOS 7. 
My Docker version is 18.06.
`$ docker --version
Docker version 18.06.1-ce, build e68fc7a`

**Comment by ericvn** [comment]
I wonder if that's too back level. I just ran things on my Mac using Docker Desktop 4.5 (Docker version 20.10.12, build e91ed57)(I do set the VM sizes higher), and it completed fine. Unfortunately, I don't have anything to verify with CentOS.

**Comment by Willem456** [comment]
@howardjohn 
When I set up the proxy, I seem to be able to connect to gcr.io but still get an error.
![image](https://user-images.githubusercontent.com/74516404/154905660-47695634-c298-4abf-98de-499624284bbd.png)

**Comment by Willem456** [comment]
By observing the execution log, I found that an error was reported when executing the command "docker buildx bake -f /work/out/linux_amd64/dockerx_build/docker-bake.json all", where the "buildx" tool requires docker version 19.03+ , and my docker version was 18.06, so I finally solved it by upgrading docker version to 20.10.12
